test/infrastructure/docker/config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/kubernetes1-226021/manager:dev
+      - image: gcr.io/k8s-staging-capi-docker/capd-manager-amd64:dev
```

Comment:
I'm fine with omitting the arch.

The reason this has the arch in the first place is that's what the make target `docker-build` will set it to. But, the e2e can execute in a dirty git environment (since `docker-build` would change `gcr.io/k8s-staging-capi-docker/capd-manager:dev` to `gcr.io/k8s-staging-capi-docker/capd-manager-amd64:dev` anyway). And literally anything is better than having a shady gcr.io bucket in the manifest 😂